GitUpdateScript.sh

```diff
@@ -94,26 +94,30 @@ rm Temp_dir_log #rm removes/dels the file
 }
```

Comment:
you can remove the RemoveFileCreatedToHoldGitReposNames function as it is no longer called